src/components/entry/EntryDetails.js

```diff
@@ -1,19 +1,253 @@
+import { Wikipedia } from '@styled-icons/boxicons-logos'
+//import { Calendar } from '@styled-icons/boxicons-regular'
+import { Flag, Map, Star } from '@styled-icons/boxicons-solid'
+import React, { useEffect, useState } from 'react'
 import { useRouteMatch } from 'react-router-dom'
+import styled from 'styled-components'
 
-const EntryDetails = () => {
+import { getLocationById, getTypeById } from '../../utils/api'
+import Button from '../ui/Button'
+import { theme } from '../ui/GlobalStyle'
+import IconButton from '../ui/IconButton'
+import { Tag } from '../ui/Tag'
+import EatTheWeedsLogo from './EatTheWeeds.svg'
+import ForagingTexasLogo from './ForagingTexas.png'
+import FruitipediaLogo from './Fruitipedia.png'
+import UrbanMushroomsLogo from './UrbanMushrooms.png'
+import USDALogo from './USDA.svg'
+
+const ACCESS_TYPE = {
+  0: "On lister's property",
+  1: 'Received permission from owner',
+  2: 'Public property',
+  3: 'Private but overhanging',
+  4: 'Private property',
+}
+
+const parseISOString = (dateString) => {
+  const date = new Date(dateString)
+
+  const options = { year: 'numeric', month: 'long', day: 'numeric' }
+  const localeDateString = date.toLocaleDateString(undefined, options)
+
+  return `${localeDateString}`
+}
+
+// Wraps the entire page and gives it a top margin if on mobile
+const EntryDetailsPageContainer = styled.div`
+  overflow: scroll;
+  margin-top: ${(props) => (props.isDesktop ? '0px' : '90px')};
+`
+
+const ImageContainer = styled.img`
+  width: 100%;
+`
+
+// Wraps all text in the container
+const EntryDetailsContent = styled.div`
+  margin: 23px;
+`
+
+const PlantName = styled.h2`
+  margin-top: 0px;
+  margin-bottom: 0px;
+  color: #333333;
+`
+
+const ScientificName = styled.small`
+  font-style: italic;
+`
+
+// Wraps the plant name and scientific name, as well as an icon button for desktop
+const HeaderContainer = styled.div`
```

Comment:
it may be possible to convert a few divs you have to a more semantic sections format (https://html.spec.whatwg.org/multipage/sections.html) -- e.g. HeaderContainer maybe could be a header, EntryDetailsContent could be an article, DescriptionContainer could potentially be a section, etc. i think that styling should be unaffected? i think i'll leave this more to your discretion, however, because:

> "The section element is not a generic container element. When an element is needed only for styling purposes or as a convenience for scripting, authors are encouraged to use the div element instead. A general rule is that the section element is appropriate only if the element's contents would be listed explicitly in the document's outline."

and mostly, your containers have to do with styling so they look good!